.github/workflows/anaconda_tests.yml

```diff
@@ -43,10 +43,11 @@ jobs:
           # - Install dependencies for blivet.
           # - Install blivet to the container.
           # - Run Anaconda tests.
-          podman run -i --rm -v ./blivet:/blivet:z -v ./anaconda:/anaconda:z quay.io/rhinstaller/anaconda-ci:$TARGET_BRANCH sh -c " \
+          podman run -i --rm -v ./blivet:/blivet:z -v ./anaconda:/anaconda:z quay.io/rhinstaller/anaconda-ci:master sh -c " \
```

Comment:
I'm afraid I don't understand why master, if this is for rhel 9?

Reply:
You don't have RHEL images (or at least I don't see one on quay). If this isn't possible, I can close the PR, I was just trying to make sure we won't break Anaconda in the future on RHEL.